src/controlPSECount.cpp

```diff
@@ -0,0 +1,20 @@
+/**
+ * @file controlPSECount.cpp
+ *
+ * @author  Guy Maurel
+ * @license GPL v2+
+ */
+
+#include "controlPSECount.h"
+
+
+void controlPSECount(size_t value)
```

Comment:
Do you intend to use that function in other files than indent.cpp? If not this could be just a static function.